src/core/utils/createApolloClient.ts

```diff
@@ -24,5 +24,21 @@ export default (host: string): ApolloClient<unknown> => {
         request: async (operation): Promise<void> => {
             await operation.setContext(getHeaders(getToken()));
         },
+        // TODO: figure out generic handler for errors
+        onError: ({ graphQLErrors = [], networkError }): void => {
+            graphQLErrors.forEach(({ message, locations, path }) =>
+                console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`),
```

Comment:
Should we be `console.log`ing in here?

Reply:
For now, yes. See the todo on line 27. i think we need to sort out error handling properly.